jaraco/develop/print-meta.py

```diff
@@ -1,9 +1,8 @@
-import autocommand
-
 from build.util import project_wheel_metadata
+from jaraco.ui.main import main
 
 
-@autocommand.autocommand(__name__)
-def main(path='.', field='Requires-Dist'):
+@main
+def run(path='.', field='Requires-Dist'):
```

Comment:
Let's annotate these?

Reply:
Sounds good. I'd do that here as a suggestion, but I don't know the syntax (where to put the spaces to satisfy the linter), so I'm going to do it outside of this PR.

Reply:
On second thought, does it make sense to annotate a variable that's assigned a static string? That feels like over-specifying to me. It's so much easier to read without the annotations and even mypy knows it's a string, right? Let's go without it for now. If there's a benefit to adding the type annotations, I'm open to hear it, but I'm more inclined to keep it simple if the typing system supports that.

I wonder how typer would handle `number = 0`; would it need the type annotation to know to construct an integer (or float?) from the input? If not, that's maybe a good justification for indicating the `str` - for symmetry and consistency with other types.

Reply:
I did end up adding the type annotation, which also gave mypy permission to start nitpicking about the `| None` of `email.Message.get_all`. Fixed in 405317a.